src/test/java/com/guysfromusa/carsgame/v1/CarApiAware.java

```diff
@@ -41,4 +41,12 @@ default Long removeCar(TestRestTemplate template, String carName){
         ResponseEntity<Long> modifiedCar =  template.exchange(url, DELETE, null, Long.class);
         return modifiedCar.getBody();
     }
+
+    default Car repairCar(TestRestTemplate template, String carName){
```

Comment:
after introducing @Sql I think we should try to load data directly to db to make tests clean and readable